packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -66,17 +66,15 @@ export async function getEmailsToInvite(usernameOrEmail: string | string[]) {
 
 export async function getUserToInviteOrThrowIfExists({
   usernameOrEmail,
-  orgId,
   isOrg,
 }: {
   usernameOrEmail: string;
-  orgId: number;
   isOrg?: boolean;
 }) {
   // Check if user exists in ORG or exists all together
   const invitee = await prisma.user.findFirst({
     where: {
-      OR: [{ username: usernameOrEmail, organizationId: orgId }, { email: usernameOrEmail }],
```

Comment:
Removing `organizationId: orgId` may not retain the right logic as [inviteMember.handler.ts#70](https://github.com/calcom/cal.com/blob/main/packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts#L70) relies on the check that the provided username is already a part of the organization.

Reply:
@InderdeepBajwa, Thank you so much for the feedback.
I updated the code. Can you please check?